Remotion/ObjectBinding/UnitTests/BusinessObjectStringFormatterServiceTests/GetPropertyString_WithDateTimeProperty.cs

```diff
@@ -71,6 +71,25 @@ public void Scalar_WithValu_AndDateProperty ()
       }
     }
 
+#if NET6_0_OR_GREATER
+    [Test]
+    public void Scalar_WithValue_AndDateOnlyProperty ()
+    {
+      _mockProperty.Setup(_ => _.Type).Returns(DateTimeType.Date).Verifiable();
+      _mockProperty.Setup(_ => _.IsList).Returns(false).Verifiable();
+      _mockBusinessObject.Setup(_ => _.GetProperty(_mockProperty.Object)).Returns(new DateOnly(2000, 6, 17)).Verifiable();
+
+      using (new CultureScope(new CultureInfo("de-de"), new CultureInfo("de-de")))
+      {
+        string actual = _stringFormatterService.GetPropertyString(_mockBusinessObject.Object, _mockProperty.Object, null);
+
+        _mockBusinessObject.Verify();
+        _mockProperty.Verify();
+        Assert.That(actual, Is.EqualTo("17.06.2000"));
+      }
+    }
+#endif
+
```

Comment:
Turns out, we can revert this since the GetProperty should return DateTime, not DateOnly. Since we are mocking the property, there is no differernce between DateTimeProperty, DateProperty and DateOnlyProperty.

In the orignal specification, the assumption was we are using actual properties loaded via reflection from the domain in the test.